src/NUnitFramework/testdata/AssertMultipleData.cs

```diff
@@ -260,6 +265,88 @@ public void AssumptionInBlock()
                 Assume.That(2 + 2 == 4);
             });
         }
+
+#if ASYNC
+        [Test]
+        public void ThreeAssertsSucceed_Async()
+        {
+            Assert.Multiple(async () =>
+            {
+                await Task.Delay(1000);
```

Comment:
I removed the async Assert.Multiple overload and reduced the times of all the delays to 100ms. All five async tests still were failing. Then I put back the async overload and they passed. I'll push and merge when CI passes.